tests/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -496,6 +496,36 @@ public function testCreateSchema(): void
         self::assertTrue($schema->hasTable('test_table'));
     }
 
+    public function testMigrateSchema(): void
+    {
+        $this->createTestTable('table_to_alter');
+        $this->createTestTable('table_to_drop');
+
+        $schema = $this->schemaManager->createSchema();
+
+        $tableToAlter = $schema->getTable('table_to_alter');
+        $tableToAlter->dropColumn('foreign_key_test');
+        $tableToAlter->addColumn('number', 'integer');
+
+        $schema->dropTable('table_to_drop');
+
+        $tableToCreate = $schema->createTable('table_to_create');
+        $tableToCreate->addColumn('id', 'integer', ['notnull' => true]);
+        $tableToCreate->setPrimaryKey(['id']);
+        $tableToCreate->addColumn('test', 'string', ['length' => 255]);
+        $tableToCreate->addColumn('foreign_key_test', 'integer');
```

Comment:
Why are the columns other than `id` are needed on this table if they are not used by the test?

Reply:
For consistency, I copied the table definition from `createTestTable()`.

But the columns that are not used can be removed indeed.